src/components/DocumentRenderer/DocumentDetailsSheet.tsx

```diff
@@ -13,16 +15,41 @@ export const DocumentDetailsSheet: FunctionComponent<DocumentDetailsSheet> = ({
   document
 }) => {
   const [headerHeight, setHeaderHeight] = useState(0);
+  const hasHeaderHeightBeenSet = useRef(false);
   const onHeaderLayout = (event: LayoutChangeEvent): void => {
-    const { height } = event.nativeEvent.layout;
-    setHeaderHeight(height + 56);
+    if (!hasHeaderHeightBeenSet.current) {
+      const { height } = event.nativeEvent.layout;
+      setHeaderHeight(height + 56);
+      hasHeaderHeightBeenSet.current = true;
+    }
   };
 
   const { name } = document.data.issuers[0];
```

Comment:
Can use `getData` from oa so that you don't have to deal with the type, salts and whatnots.